### ISSUE TITLE
PUSH_BUTTON_FLAT:  don't get chanelx.ON events

### ISSUE THREAD

**Issue by jmades** [iobroker-community-adapters/ioBroker.hmip#485]
The PUSH_BUTTON_FLAT device is correctly shown in the object tree, providing 3 channels, two of them (1+2) functionalChannelTypes (see attachment ![hmip](https://github.com/iobroker-community-adapters/ioBroker.hmip/assets/65286060/785cb3c6-5dcb-4ae6-8f88-e3bb611d9bcc)). 

Unfortunately pressing the keys of the push button I do not get an event.

Unexpected: When changing the configuration of the push button with the HMIP app, both events are fired when updating configuration by pressing the system button. 

************* Script code **************************
const taster_ch1 = 'hmip.0.devices.3014F711A0002C1D899CE0B7.channels.1.on';
const taster_ch2 = 'hmip.0.devices.3014F711A0002C1D899CE0B7.channels.2.on';

on({id : taster_ch1}, function (obj) 
    {
        console.log("Ch1 ON")
    }
)

on({id : taster_ch2}, function (obj) 
    {
        console.log("Ch2 ON")
    }
)

**Comment by jmades** [comment]
Adapter version: 1.20.0

**Comment by Apollon77** [comment]
Yes, like in homematic push buttons are normally not pushed by eq-3 as info, so no chance for the adapter.sorry

**Comment by jmades** [comment]
@Apollon77 could you please give me short Informationen about what eq-3 is?
I'm always interested to understand better the technology behind the adapter

**Comment by Apollon77** [comment]
eq-3 is the company that produces all HMIP and Homematic devices :-)

**Comment by Apollon77** [comment]
Waaaiitt ... you are aware that all that we do uses inofficialö APIS opf the HMIP App? So ... better not complain at eq-3 :-)

**Comment by jmades** [comment]
Ok, I didn't know that.Then I will have to accept that and search for any other switch helping me to realise my project. Any proposal for switches just firing keypress events to iobroker?Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Ingo Fischer ***@***.***> Datum: 01.12.23  12:29  (GMT+01:00) An: "iobroker-community-adapters/ioBroker.hmip" ***@***.***> Cc: jmades ***@***.***>, Author ***@***.***> Betreff: Re: [iobroker-community-adapters/ioBroker.hmip] PUSH_BUTTON_FLAT:  don't get channel x.ON events (Issue #485) 
Waaaiitt ... you are aware that all that we do uses inofficialö APIS opf the HMIP App? So ... better not complain at eq-3 :-)
